packages/core/src/envelope.ts

```diff
@@ -86,3 +89,31 @@ export function createEventEnvelope(
   const eventItem: EventItem = [{ type: eventType }, event];
   return createEnvelope<EventEnvelope>(envelopeHeaders, [eventItem]);
 }
+
+/**
+ * Create an Envelope from an event.
+ */
+export function createAttachmentEnvelope(
```

Comment:
we need to send the attachment in a seperate envelope

Reply:
hmm, this is a bit weird IMHO. We are not really creating an attachment envelope, aren't we? We are just sending a regular (?) event and adding attachments to it...? Or am I misunderstanding this? What is `event` in this case?

Could we not just use `client.sendEvent()` here - that takes a `hint` which will add any attachments from it. So we could do e.g.:

```js
client.sendEvent(feedbackEvent, { attachments });
```

This would also take care of all the metadata etc.

Reply:
afaik there's something in the backend that's preventing this from working properly, because the old user-feedbacks had their own pipeline/queue or something. 

@andrewshie-sentry is working on creating a new queue that'll fix the issue. maybe @JoshFerge can also explain in the meantime. It's really causing a bunch of extra code in here to manage the attachment sending :(

Reply:
yeah, but we can still send it twice? we are already doing this, should have the same outcome I believe?

Reply:
yeah, we need to separate out some things in relay / our own consumer long term for this to work well.